app/services/conversation_bot.rb

```diff
@@ -274,7 +274,7 @@ def handle_location(from_or_to)
           @response = I18n.t(:no_address_match, locale: @locale)
           return @bot_counter + 1
         end
-
+        
```

Comment:
[editorconfig](https://editorconfig.org/) would help keep whitespace changes to a minimum

Reply:
great suggestion, added a .editorconfig file